en/config_mc/pid_tuning_guide_multicopter_basic.md

```diff
@@ -0,0 +1,93 @@
+# Multicopter PID Tuning Guide
+
+This tutorial explains how to tune the PID loops on PX4 for all [multicopter setups](../airframes/airframe_reference.md#copter) (Quads, Hexa, Octo etc).
+
+Tuning is recommended for all new vehicle setups, because relatively small hardware and assembly changes can affect the gains required tuning gains for optimal flight.
+For example, different ESCs or motors require different tuning gains.
+
+:::tip
+Generally if you're using an appropriate [supported airframe configuration](../airframes/airframe_reference.md#copter) (selected from [QGroundControl > Airframe](../config/airframe.md)), the default tuning should allow you to fly the vehicle safely.
+To get the _very best_ performance we recommend you tune your new vehicle.
+:::
+
+:::warning
+Poorly tuned vehicles are likely to be unstable, and easy to crash.
+Make sure to have assigned a [Kill switch](../config/safety.md#emergency-switches).
+:::
+
+## Introduction
+
+PX4 uses **P**roportional, **I**ntegral, **D**erivative (PID) controllers (these are the most widespread control technique).
```

Comment:
@bkueng This got a significant update. 

This talked about need for tuning the controllers in order, which of felt out of place. I fixed this by moving up the information about the tuning UI and the general remarks. So this is more of a true overview.

I did decide to leave the guidance on how you modify PID down in the procedure.

The general remarks are still _embedded_ in the procedure.

Reply:
Makes sense, more integrated now